.github/workflows/scheduled-cleanup.yml

```diff
@@ -0,0 +1,125 @@
+# Clean-up AWS accounts on a CRON Schedule
+name: Clean AWS accounts
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch.
+on:
+  schedule:
+    # Runs "daily at 6:15am UTC - 11:15pm PDT - 1:15am CDT - 8:15am CEST"
+    # Github Actions have heavy load every start hour, so we wait 15 min to avoid delays in our jobs
+    - cron: '15 6 * * *'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      region:
+        description: 'Optional region to delete'
+
+jobs:
+  setup-module-cleanup:
+    runs-on: ubuntu-latest
+    outputs:
+      set-test-module-names: ${{ steps['set-test-module-names'].outputs['test-module-names'] }}
+    steps:
+      - uses: actions/checkout@v2
+      - run: yarn
+      - id: set-test-module-names
+        name: Set modules tests names
+        run: echo "::set-output name=test-module-names::$(npx jest **/modules/* --listTests --json | jq -c 'map(split("/") | .[-1] | split(".") | .[0] | gsub( "-"; "_") | ascii_upcase | tostring)')"
+
+  # Job by region since we pass the 256 job limit running them all at once
```

Comment:
just a note that if AWS adds regions we need to update a few things manually in our codebase